platform/log/zephyr/src/mender-log.c

```diff
@@ -33,36 +33,47 @@ mender_log_init(void) {
     return MENDER_OK;
 }
 
+/* IMPORTANT: Please note that the default size of internal Zephyr logger buffer is 1024 bytes; if we need log buffer size 
+              to be more than that there is CONFIG_LOG_BUFFER_SIZE option allowing to extend the buffer size. */
+#define LOG_BUFFER_SIZE_BYTES (128)
```

Comment:
What are we trying to achieve by harcoding a number here at all, then.

The configurable value is from 128 to 65536, which means that for a user configuring it for the minimum, we won't be able to fit our logs. Either we have an explicit requirement of the logging buffer size of we _adapt_ our buffer size to `CONFIG_LOG_BUFFER_SIZE` minus the length for our prefix "(line": ".

Reply:
Not sure if I get it. We talking about two different buffers here. One is for printing messages (which is hard-coded now to `LOG_BUFFER_SIZE_BYTES (128)`). It means that if you have a log that is longer than 128 characters it will be truncated. 
The other buffer of size `CONFIG_LOG_BUFFER_SIZE` is for accumulating log messages and is Zephyr cyclic log buffer. The comment is mainly to make sure that the `LOG_BUFFER_SIZE_BYTES < CONFIG_LOG_BUFFER_SIZE`.

Reply:
Okey, I didn't get that internal buffer was a circular buffer.

Nevermind, then.